src/ToolUpdate/UpdateExecutor.php

```diff
@@ -125,6 +142,9 @@ private function deleteVersion(ToolInformationInterface $tool): void
         if (!$bootstrap instanceof InstalledBootstrap) {
             throw new RuntimeException('Can only remove installed bootstrap files.');
         }
+        if ($signatureUrl = $tool->getSignatureUrl()) {
```

Comment:
Please do not separate the type check of bootstrap from its actual deletion.
```php
if (!$bootstrap instanceof InstalledBootstrap) {
    throw new RuntimeException('Can only remove installed bootstrap files.');
}
$this->deleteFile($bootstrap->getFilePath());
```
belongs together, so code here should be:
```php
if (!$bootstrap instanceof InstalledBootstrap) {
    throw new RuntimeException('Can only remove installed bootstrap files.');
}
$this->deleteFile($bootstrap->getFilePath());

if ($signatureUrl = $tool->getSignatureUrl()) {
    $this->deleteFile($this->phpcqPath . '/' . $signatureUrl);
}
```

Reply:
Done with 42a95b6.